superset/assets/src/dashboard/deprecated/v1/components/Header.jsx

```diff
@@ -43,25 +43,11 @@ class Header extends React.PureComponent {
       showV2PromptModal: props.dashboard.promptV2Conversion,
     };
     this.toggleShowV2PromptModal = this.toggleShowV2PromptModal.bind(this);
-    this.handleConvertToV2 = this.handleConvertToV2.bind(this);
+    this.handleConvertToV2 = props.handleConvertToV2.bind(this);
   }
   handleSaveTitle(title) {
     this.props.updateDashboardTitle(title);
   }
-  handleConvertToV2(editMode) {
```

Comment:
here, `handleConvertToV2` is refactored out into Dashboard component, since this function is called from both Dashboard and Header component.